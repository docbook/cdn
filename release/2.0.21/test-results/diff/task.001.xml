<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <figure deltaxml:deltaV2="A" id="R.1.2" class="task-wrapper">
            <div class="task">
               <figure id="R.1.2.1" class="tasksummary-wrapper">
                  <div class="tasksummary">
                     <p>This task doesn’t accomplish anything.</p>
                  </div>
               </figure>
               <figure id="R.1.2.2" class="taskprerequisites-wrapper">
                  <div class="taskprerequisites">
                     <p>Do something else first.</p>
                  </div>
               </figure>
               <figure id="R.1.2.3" class="procedure-wrapper">
                  <div class="procedure">
                     <ol type="1">
                        <li class="step">
                           <div>
                              <p>Do This</p>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do That</p>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do one of these things:</p>
                              <div class="stepalternatives">
                                 <ul>
                                    <li class="step">
                                       <div>
                                          <p>Do A</p>
                                       </div>
                                    </li>
                                    <li class="step">
                                       <div>
                                          <p>Do B</p>
                                       </div>
                                    </li>
                                    <li class="step">
                                       <div>
                                          <p>Do C</p>
                                       </div>
                                    </li>
                                 </ul>
                              </div>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do the other thing</p>
                           </div>
                        </li>
                     </ol>
                  </div>
               </figure>
               <figure id="R.1.2.4" class="taskrelated-wrapper">
                  <div class="taskrelated">
                     <p>See some more interesting task after you’ve done this one.</p>
                  </div>
               </figure>
            </div>
         </figure><div deltaxml:deltaV2="B" class="content">
            <figure id="R.1.2" class="task-wrapper">
               <div class="task">
                  <figure id="R.1.2.1" class="tasksummary-wrapper">
                     <div class="tasksummary">
                        <p>This task doesn’t accomplish anything.</p>
                     </div>
                  </figure>
                  <figure id="R.1.2.2" class="taskprerequisites-wrapper">
                     <div class="taskprerequisites">
                        <p>Do something else first.</p>
                     </div>
                  </figure>
                  <figure id="R.1.2.3" class="procedure-wrapper">
                     <div class="procedure">
                        <ol type="1">
                           <li class="step">
                              <div>
                                 <p>Do This</p>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do That</p>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do one of these things:</p>
                                 <div class="stepalternatives">
                                    <ul>
                                       <li class="step">
                                          <div>
                                             <p>Do A</p>
                                          </div>
                                       </li>
                                       <li class="step">
                                          <div>
                                             <p>Do B</p>
                                          </div>
                                       </li>
                                       <li class="step">
                                          <div>
                                             <p>Do C</p>
                                          </div>
                                       </li>
                                    </ul>
                                 </div>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do the other thing</p>
                              </div>
                           </li>
                        </ol>
                     </div>
                  </figure>
                  <figure id="R.1.2.4" class="taskrelated-wrapper">
                     <div class="taskrelated">
                        <p>See some more interesting task after you’ve done this one.</p>
                     </div>
                  </figure>
               </div>
            </figure>
            <figure id="R.1.3" class="task-wrapper">
               <header class="task-titlepage">
                  <h3>A Task</h3>
               </header>
               <div class="task">
                  <figure id="R.1.3.2" class="tasksummary-wrapper">
                     <div class="tasksummary">
                        <p>This task doesn’t accomplish anything.</p>
                     </div>
                  </figure>
                  <figure id="R.1.3.3" class="taskprerequisites-wrapper">
                     <div class="taskprerequisites">
                        <p>Do something else first.</p>
                     </div>
                  </figure>
                  <figure id="R.1.3.4" class="procedure-wrapper">
                     <div class="procedure">
                        <ol type="1">
                           <li class="step">
                              <div>
                                 <p>Do This</p>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do That</p>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do one of these things:</p>
                                 <div class="stepalternatives">
                                    <ul>
                                       <li class="step">
                                          <div>
                                             <p>Do A</p>
                                          </div>
                                       </li>
                                       <li class="step">
                                          <div>
                                             <p>Do B</p>
                                          </div>
                                       </li>
                                       <li class="step">
                                          <div>
                                             <p>Do C</p>
                                          </div>
                                       </li>
                                    </ul>
                                 </div>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do the other thing</p>
                              </div>
                           </li>
                        </ol>
                     </div>
                  </figure>
                  <figure id="R.1.3.5" class="taskrelated-wrapper">
                     <div class="taskrelated">
                        <p>See some more interesting task after you’ve done this one.</p>
                     </div>
                  </figure>
               </div>
            </figure>
            <figure id="R.1.4" class="task-wrapper">
               <header class="task-titlepage">
                  <h3>A Task</h3>
               </header>
               <div class="task">
                  <figure id="R.1.4.2" class="tasksummary-wrapper">
                     <div class="tasksummary">
                        <p>This task doesn’t accomplish anything.</p>
                     </div>
                  </figure>
                  <figure id="R.1.4.3" class="taskprerequisites-wrapper">
                     <h4>Prerequisites</h4>
                     <div class="taskprerequisites">
                        <p>Do something else first.</p>
                     </div>
                  </figure>
                  <figure id="R.1.4.4" class="procedure-wrapper">
                     <h4>Procedure</h4>
                     <div class="procedure">
                        <ol type="1">
                           <li class="step">
                              <div>
                                 <p>Do This</p>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do That</p>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do one of these things:</p>
                                 <div class="stepalternatives">
                                    <ul>
                                       <li class="step">
                                          <div>
                                             <p>Do A</p>
                                          </div>
                                       </li>
                                       <li class="step">
                                          <div>
                                             <p>Do B</p>
                                          </div>
                                       </li>
                                       <li class="step">
                                          <div>
                                             <p>Do C</p>
                                          </div>
                                       </li>
                                    </ul>
                                 </div>
                              </div>
                           </li>
                           <li class="step">
                              <div>
                                 <p>Do the other thing</p>
                              </div>
                           </li>
                        </ol>
                     </div>
                  </figure>
                  <figure id="R.1.4.5" class="taskrelated-wrapper">
                     <h4>Related</h4>
                     <div class="taskrelated">
                        <p>See some more interesting task after you’ve done this one.</p>
                     </div>
                  </figure>
               </div>
            </figure>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.3" class="task-wrapper">
            <header class="task-titlepage">
               <h3>A Task</h3>
            </header>
            <div class="task">
               <figure id="R.1.3.2" class="tasksummary-wrapper">
                  <div class="tasksummary">
                     <p>This task doesn’t accomplish anything.</p>
                  </div>
               </figure>
               <figure id="R.1.3.3" class="taskprerequisites-wrapper">
                  <div class="taskprerequisites">
                     <p>Do something else first.</p>
                  </div>
               </figure>
               <figure id="R.1.3.4" class="procedure-wrapper">
                  <div class="procedure">
                     <ol type="1">
                        <li class="step">
                           <div>
                              <p>Do This</p>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do That</p>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do one of these things:</p>
                              <div class="stepalternatives">
                                 <ul>
                                    <li class="step">
                                       <div>
                                          <p>Do A</p>
                                       </div>
                                    </li>
                                    <li class="step">
                                       <div>
                                          <p>Do B</p>
                                       </div>
                                    </li>
                                    <li class="step">
                                       <div>
                                          <p>Do C</p>
                                       </div>
                                    </li>
                                 </ul>
                              </div>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do the other thing</p>
                           </div>
                        </li>
                     </ol>
                  </div>
               </figure>
               <figure id="R.1.3.5" class="taskrelated-wrapper">
                  <div class="taskrelated">
                     <p>See some more interesting task after you’ve done this one.</p>
                  </div>
               </figure>
            </div>
         </figure><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.4" class="task-wrapper">
            <header class="task-titlepage">
               <h3>A Task</h3>
            </header>
            <div class="task">
               <figure id="R.1.4.2" class="tasksummary-wrapper">
                  <div class="tasksummary">
                     <p>This task doesn’t accomplish anything.</p>
                  </div>
               </figure>
               <figure id="R.1.4.3" class="taskprerequisites-wrapper">
                  <h4>Prerequisites</h4>
                  <div class="taskprerequisites">
                     <p>Do something else first.</p>
                  </div>
               </figure>
               <figure id="R.1.4.4" class="procedure-wrapper">
                  <h4>Procedure</h4>
                  <div class="procedure">
                     <ol type="1">
                        <li class="step">
                           <div>
                              <p>Do This</p>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do That</p>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do one of these things:</p>
                              <div class="stepalternatives">
                                 <ul>
                                    <li class="step">
                                       <div>
                                          <p>Do A</p>
                                       </div>
                                    </li>
                                    <li class="step">
                                       <div>
                                          <p>Do B</p>
                                       </div>
                                    </li>
                                    <li class="step">
                                       <div>
                                          <p>Do C</p>
                                       </div>
                                    </li>
                                 </ul>
                              </div>
                           </div>
                        </li>
                        <li class="step">
                           <div>
                              <p>Do the other thing</p>
                           </div>
                        </li>
                     </ol>
                  </div>
               </figure>
               <figure id="R.1.4.5" class="taskrelated-wrapper">
                  <h4>Related</h4>
                  <div class="taskrelated">
                     <p>See some more interesting task after you’ve done this one.</p>
                  </div>
               </figure>
            </div>
         </figure>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>