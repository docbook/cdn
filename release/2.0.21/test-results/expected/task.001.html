<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><figure id="R.1.2" class="task-wrapper"><div class="task">
<figure id="R.1.2.1" class="tasksummary-wrapper"><div class="tasksummary">
<p>This task doesn’t accomplish anything.</p>
</div></figure>
<figure id="R.1.2.2" class="taskprerequisites-wrapper"><div class="taskprerequisites">
<p>Do something else first.</p>
</div></figure>
<figure id="R.1.2.3" class="procedure-wrapper"><div class="procedure"><ol type="1"><li class="step"><div><p>Do This</p></div></li><li class="step"><div><p>Do That</p></div></li><li class="step"><div><p>Do one of these things:</p><div class="stepalternatives"><ul>
<li class="step"><div><p>Do A</p></div></li>
<li class="step"><div><p>Do B</p></div></li>
<li class="step"><div><p>Do C</p></div></li>
</ul></div></div></li><li class="step"><div><p>Do the other thing</p></div></li></ol></div></figure>
<figure id="R.1.2.4" class="taskrelated-wrapper"><div class="taskrelated">
<p>See some more interesting task after you’ve done this one.</p>
</div></figure>
</div></figure><figure id="R.1.3" class="task-wrapper"><header class="task-titlepage"><h3>A Task</h3></header><div class="task">

<figure id="R.1.3.2" class="tasksummary-wrapper"><div class="tasksummary">
<p>This task doesn’t accomplish anything.</p>
</div></figure>
<figure id="R.1.3.3" class="taskprerequisites-wrapper"><div class="taskprerequisites">
<p>Do something else first.</p>
</div></figure>
<figure id="R.1.3.4" class="procedure-wrapper"><div class="procedure"><ol type="1"><li class="step"><div><p>Do This</p></div></li><li class="step"><div><p>Do That</p></div></li><li class="step"><div><p>Do one of these things:</p><div class="stepalternatives"><ul>
<li class="step"><div><p>Do A</p></div></li>
<li class="step"><div><p>Do B</p></div></li>
<li class="step"><div><p>Do C</p></div></li>
</ul></div></div></li><li class="step"><div><p>Do the other thing</p></div></li></ol></div></figure>
<figure id="R.1.3.5" class="taskrelated-wrapper"><div class="taskrelated">
<p>See some more interesting task after you’ve done this one.</p>
</div></figure>
</div></figure><figure id="R.1.4" class="task-wrapper"><header class="task-titlepage"><h3>A Task</h3></header><div class="task">

<figure id="R.1.4.2" class="tasksummary-wrapper"><div class="tasksummary">
<p>This task doesn’t accomplish anything.</p>
</div></figure>
<figure id="R.1.4.3" class="taskprerequisites-wrapper"><h4>Prerequisites</h4><div class="taskprerequisites">

<p>Do something else first.</p>
</div></figure>
<figure id="R.1.4.4" class="procedure-wrapper"><h4>Procedure</h4><div class="procedure"><ol type="1"><li class="step"><div><p>Do This</p></div></li><li class="step"><div><p>Do That</p></div></li><li class="step"><div><p>Do one of these things:</p><div class="stepalternatives"><ul>
<li class="step"><div><p>Do A</p></div></li>
<li class="step"><div><p>Do B</p></div></li>
<li class="step"><div><p>Do C</p></div></li>
</ul></div></div></li><li class="step"><div><p>Do the other thing</p></div></li></ol></div></figure>
<figure id="R.1.4.5" class="taskrelated-wrapper"><h4>Related</h4><div class="taskrelated">

<p>See some more interesting task after you’ve done this one.</p>
</div></figure>
</div></figure></article><script src="../../resources/js/prism.js"></script></body></html>